integrationExamples/gpt/userId_example.html

```diff
@@ -83,9 +83,10 @@
         var adUnits = [
             {
                 code: 'test-div',
-                sizes: [[300,250],[300,600],[728,90]],
                 mediaTypes: {
-                    banner: {}
+                    banner: {
+                        sizes: [[300,250],[300,600],[728,90]]
```

Comment:
I noticed that a recent PR tackled the same issue (https://github.com/prebid/Prebid.js/pull/5606), but I was still getting errors if the sizes aren't defined inside the _banner_ object. I can move this to a different PR if that'd be better.